packages/swap/src/handlers/__tests__/__snapshots__/openSwapDepositChannel.test.ts.snap

```diff
@@ -281,6 +281,7 @@ exports[`openSwapDepositChannel > creates channel with ccmParams and stores it i
   [
     {
       "ccmParams": {
+        "additionalData": undefined,
```

Comment:
the output type of the `ccmParamsSchema` has a required `additionalData` key now because of the fallback